ios/Plugin/Plugin.m

```diff
@@ -11,6 +11,7 @@
            CAP_PLUGIN_METHOD(addMarker, CAPPluginReturnPromise);
            CAP_PLUGIN_METHOD(addMarkers, CAPPluginReturnPromise);
            CAP_PLUGIN_METHOD(removeMarker, CAPPluginReturnPromise);
+           CAP_PLUGIN_METHOD(removeMap, CAPPluginReturnPromise);
```

Comment:
Can you move this line to line 10 instead (right below `updateMap`)?

Reply:
resolved